### DISCUSSION TITLE
Examples of using other embeddding rather than OpenAI/Huggingface?

### DISCUSSION THREAD

**Discussion by orchardc** [microsoft/semantic-kernel#2989] [Q&A]
Do we have examples of how to use other embeddings (such as "Sentence Transformer Embeddings")  rather than OpenAI/Huggingface? 
```
        var kernel = Kernel.Builder
            .WithOpenAIChatCompletionService(OpenAI_ChatModelId, OpenAI_ApiKey)
            .WithOpenAITextEmbeddingGenerationService(OpenAI_EmbeddingModelId, OpenAI_ApiKey)
            .Build();
```

**Comment by dmytrostruk** [comment]
Hi @orchardc !
It's possible to implement your own LLM service and inject it into Kernel like in this example:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/KernelSyntaxExamples/Example16_CustomLLM.cs

Please let me know if this covers your scenario. Thank you!
